jmclient/jmclient/blockchaininterface.py

```diff
@@ -438,7 +445,8 @@ def estimate_fee_per_kb(self, N):
 
         if retval == -1:
             retval = int(Decimal(1e8) * Decimal(estimate))
-        log.info("Using tx fee: " + btc.fee_per_kb_to_str(retval))
+        log.info("Using bitcoin network feerate (per kB in sats): " +\
```

Comment:
```suggestion
        log.info("Using bitcoin network feerate: " +\
```
`fee_per_kb_to_str()` output units too.

Reply:
Agreed, done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/750/commits/8cdf3e0a4ac218327fb9928d81273b2ca311fc12